### PR TITLE
update project description in gemspec

### PR DESCRIPTION
**Pull request by mojavelinux**
I noticed that the project description in the gem was outdated. I tried to come up with a way to encapsulate all this library does in a single sentence. It may still need some work. Feedback welcome. cc: @djencks @Mogztter

### CODE REVIEW

File: asciidoctor-diagram.gemspec [hunk 1]
```diff
@@ -9,7 +9,7 @@ Gem::Specification.new do |s|
   s.authors       = ['Pepijn Van Eeckhoudt']
   s.email         = ['pepijn@vaneeckhoudt.net']
   s.description   = %q{Asciidoctor diagramming extension}
-  s.summary       = %q{An extension for asciidoctor that adds support for UML diagram generation using PlantUML}
+  s.summary       = %q{A set of Asciidoctor extensions that generate a broad range of diagrams from plain text representations encoded in your AsciiDoc document, including integration with PlantUML, ditaa, and kroki}
```
Comment (Mogztter) [inline asciidoctor-diagram.gemspec:12]:
```suggestion
  s.summary       = %q{A set of Asciidoctor extensions that generate a broad range of diagrams from plain text representations encoded in your AsciiDoc document, including integration with PlantUML, ditaa, and Kroki}
```

**Comment by Mogztter** [comment]
> An extension for Asciidoctor that produces diagrams from plain text source using PlantUML, ditaa, Kroki, and other generators.

I agree, we probably need to add "and others" otherwise it might sound like Asciidoctor Diagram supports PlantUML, ditaa, Kroki and that's it.

**Comment by mojavelinux** [comment]
> I agree, we probably need to add "and others" otherwise it might sound like Asciidoctor Diagram supports PlantUML, ditaa, Kroki and that's it.

"including" implies there are others, but I also recognize that with a global audience, we should be explicit as possible.

**Comment by mojavelinux** [comment]
> There are blockMacro processors as well as block processors, so the source doesn't have to be in your AsciiDoc document.

I think that's getting to nuanced. Regardless of how the source is included, it's still included. I'll see if I can find a middle ground.

**Comment by mojavelinux** [comment]
I've made updates. Let me know what you think.

**Comment by pepijnve** [comment]
> A family of Asciidoctor extensions for generating a broad range of diagrams sourced from plain text embedded in or referenced by your AsciiDoc document, including integration with PlantUML, ditaa, Kroki, and many others

Condensing that a bit, cutting out superfluous words I came up with this. 'Descriptions' is the one word I'm not 100% comfortable with. It's "just" the project description, so I don't think we need to overthink this either 😄 

A family of Asciidoctor extensions that generate images from a broad range of embedded plain text diagram descriptions, including PlantUML, ditaa, Kroki, and many others.

**Comment by mojavelinux** [comment]
Updated.

If you aren't comfortable with "descriptions", might I offer "representations" as an alternative?

**Comment by mojavelinux** [comment]
One other idea is "depictions", which fits because you truly are describing how you want it to look.

**Comment by pepijnve** [comment]
It's all good for me Dan, just sharing my 2c here. The diagram code is declarative indeed. As you said "you are describing what you want". If you state it that way description seems like the appropriate noun after all.

**Comment by mojavelinux** [comment]
Excellent. Merge at will :+1:

**Comment by pepijnve** [comment]
Wish granted!

**Pull request merged**